application/config/internal.php

```diff
@@ -156,7 +156,7 @@
             'noCsrfValidationParams' => array(),
             'noCsrfValidationRoutes' => array(
                 'rest',
-                'remotecontrol',
```

Comment:
It feels like a breaking change to expect users to update this if they have overridden this in their config, so I wonder if the maintainers feel strongly about relaxing the regex in `application/core/LSHttpRequest.php` instead.

Reply:
surely if a user override this, they extend the array, not replace it

Reply:
> It feels like a breaking change to expect users to update this if they have overridden this in their config, so I wonder if the maintainers feel strongly about relaxing the regex in `application/core/LSHttpRequest.php` instead.

The commit was done to be sure to allow only needed route :). Relaxing : less CRSF control. I think we **must** keep current regexp.

Reply:
yeah I agree, it needs to stay as the more restrictive regexp

the issue before is that routes that match a portion of the routes exempted from csrf checking would also be exempt

ie 
1. rest/v1/session
2. admin/menus/sa/restore

both are exempted because they contain "rest" anywhere

this is definitely not desirable for security